src/App.jsx

```diff
@@ -1,33 +1,24 @@
-import React, { useEffect } from "react"
+import React from "react"
 import "./styles/app.css"
-import { BrowserRouter, Route, Routes, Link } from "react-router-dom"
-import { db } from "./services/firebase"
```

Comment:
Do we not need firebase in the App?

Reply:
Yes we do, that db const is the firebase. If you mean the firebase authentication, we don't need that because I remember the PO didn't mention anything about username or such. She wanted this as simple as possible I guess

Reply:
App component should only has these lines:

```
import React from "react"
import "./styles/app.css"
import { BrowserRouter, Route, Routes } from "react-router-dom"

import SessionProvider from "./contexts/SessionContext"
import Onboarding from "./views/Onboarding"
import Rules from "./views/Rules"
import Game from "./views/Game"
```